README.md

```diff
@@ -138,9 +138,14 @@ To configure OpenSSH server to fetch users’ authorized keys from LDAP server:
     `/usr/bin` with owner `root` and mode `0755`.
 2.  Add these two lines to /etc/ssh/sshd_config:
 
+
         AuthorizedKeysCommand /usr/bin/ssh-ldap-pubkey-wrapper
         AuthorizedKeysCommandUser nobody
 
+> Note: In case you install `ssh-ldap-key` from pip, `AuthorizedKeysCommand` must be:
```

Comment:
This is not true, it depends on a system/distribution – it may be `/usr/bin`, `/usr/local/bin` or even anything else. However it’s a good idea to add some note that user should check location of `ssh-ldap-pubkey-wrapper` and adjust the config accordingly.